.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "files.eol": "\n"
+}
```

Comment:
- [ ] [OPTIONAL] If you ignore this file to `.gitignore` it would be better 😃

Reply:
Sure we will write it to .gitignore. Thanks for valuable suggestion.